tests/unit/test_io_convert.py

```diff
@@ -259,6 +282,7 @@ class MixinTestZarrToZarr():
                     DirectoryStore('test_export_DirectoryStore_Export.zarr'),
                     TempStore(dir=os.path.dirname(__file__)),   # set dir to avoid switching drives on Windows
                     NestedDirectoryStore('test_export_NestedDirectoryStore_Export.zarr')]
+    TARGET_FORMAT = "Zarr"
```

Comment:
Same as above. I believe this needs to be "ZARR" (all caps) for the checks for references to be run.

```suggestion
    TARGET_FORMAT = "ZARR"
```